_integration_tests/test/notifications/activity_notifications.js

```diff
@@ -50,70 +50,70 @@ describe('Consumer API:   Receive global CallActivity Notifications', () => {
+        const messageIsAboutCorrectActivity = activityReachedMessage.flowNodeId === sampleActivityMessage.flowNodeId;
 
-        should(messageIsAboutCorrectCallActivity).be.true();
+        should(messageIsAboutCorrectActivity).be.true();
 
         resolve();
       };
 
       const subscribeOnce = true;
       await testFixtureProvider
         .consumerApiClientService
-        .onCallActivityWaiting(defaultIdentity, notificationReceivedCallback, subscribeOnce);
+        .onActivityReached(defaultIdentity, notificationReceivedCallback, subscribeOnce);
 
       await processInstanceHandler.startProcessInstanceAndReturnCorrelationId(processModelId, correlationId);
     });
   });
 
-  it('should send a notification via socket when a CallActivity is finished', async () => {
+  it('should send a notification via socket when an Activity is finished', async () => {
 
     return new Promise(async (resolve, reject) => {
 
       let notificationReceived = false;
 
-      const notificationReceivedCallback = async (callActivityFinishedMessage) => {
-        should(callActivityFinishedMessage).not.be.undefined();
-        should(callActivityFinishedMessage.flowNodeId).be.equal(callActivityToFinish.flowNodeId);
-        should(callActivityFinishedMessage.processModelId).be.equal(callActivityToFinish.processModelId);
-        should(callActivityFinishedMessage.correlationId).be.equal(callActivityToFinish.correlationId);
+      const notificationReceivedCallback = async (activityFinishedMessage) => {
+        should(activityFinishedMessage).not.be.undefined();
+        should(activityFinishedMessage.flowNodeId).be.equal(activityToFinish.flowNodeId);
+        should(activityFinishedMessage.processModelId).be.equal(activityToFinish.processModelId);
+        should(activityFinishedMessage.correlationId).be.equal(activityToFinish.correlationId);
         notificationReceived = true;
       };
 
       const subscribeOnce = true;
       await testFixtureProvider
         .consumerApiClientService
-        .onCallActivityFinished(defaultIdentity, notificationReceivedCallback, subscribeOnce);
+        .onActivityFinished(defaultIdentity, notificationReceivedCallback, subscribeOnce);
 
       const processFinishedCallback = () => {
         if (!notificationReceived) {
-          throw new Error('Did not receive the expected notification about the finished CallActivity!');
+          throw new Error('Did not receive the expected notification about the finished Activity!');
         }
         resolve();
       };
       processInstanceHandler.waitForProcessInstanceToEnd(correlationId, processModelId, processFinishedCallback);
     });
   });
 
-  it('should fail to subscribe for the CallActivityWaiting notification, if the user is unauthorized', async () => {
+  it('should fail to subscribe for the ActivityWaiting notification, if the user is unauthorized', async () => {
```

Comment:
Ist, glaube ich, ein Typo:

```suggestion
  it('should fail to subscribe for the ActivityReached notification, if the user is unauthorized', async () => {
```

Reply:
Yep